package.json

```diff
@@ -42,6 +42,7 @@
     "chalk": "^4.1",
     "commander": "^9.0",
     "debug": "^4.3.3",
+    "dotenv": "^16.0.1",
```

Comment:
Since OS env vars are not automatically injected into your node env, the following screenshots show the problem with removing `dotenv` package (ignore the white square where I censor my addresses lol).

1. With `dotenv` removed and using an OS env var either set from an `export` or added to `~/.bashrc`
![env1](https://user-images.githubusercontent.com/2993386/179051669-555eb620-3ef2-40b3-92fb-7f8c737b6a1c.png)

2. With `dotenv` removed but env var injected at run time
![env2](https://user-images.githubusercontent.com/2993386/179051939-1594ef13-cbe2-4299-8974-f498574c21de.png)